src/Credits/index.js

```diff
@@ -40,8 +40,10 @@ class Credits extends Component {
         secondaryReviewers.push(reviewer.name)
       })
 
+    const creditsClass = 'hw-structured-content-credits'
+
     return (
```

Comment:
there was no default prop here so it was coming back with undefined, or when supplied with an empty string it was preceded by an annoying space character.